src/timesteppers/forward_euler.jl

```diff
@@ -8,29 +8,17 @@ Simple forward Euler time stepping scheme.
     Δt::NF = 300.0
```

Comment:
We can do this in another PR, but from a user perspective it would nice to also allow `Period` inputs for `Δt`.